core/variable_operations.cpp

```diff
@@ -277,6 +277,20 @@ Variable sqrt(const VariableConstProxy &var) {
   return transform<double, float>(var, [](const auto x) { return sqrt(x); });
 }
 
+Variable sqrt(Variable &&var) {
+  using std::sqrt;
```

Comment:
Is this needed?

Reply:
Yes, for ADL. If below `y` is a `double` it will use `std::sqrt`, if it is `ValueAndVariance<double>` it will use the `sqrt` define for that.